apps/trading/e2e/tests/deal_ticket/test_trading_deal_ticket_submit_account.py

```diff
@@ -47,5 +47,5 @@ def test_should_show_an_error_if_your_balance_is_zero(continuous_market, vega: V
     # 7002-SORD-060
     expect(page.get_by_test_id(place_order)).to_be_enabled()
     # 7002-SORD-003
-    expect(page.get_by_test_id("deal-ticket-error-message-zero-balance")).to_have_text("You need tDAI in your wallet to trade in this market. Make a deposit")
+    expect(page.get_by_test_id("deal-ticket-error-message-zero-balance")).to_have_text("You need tDAI in your wallet to trade in this market.Make a deposit")
```

Comment:
ooh weird. How does the lack of this white space look in the ui out of interest, is it noticeable?

Reply:
Different lines so the space isn't required

Reply:
nice.